### DISCUSSION TITLE
INSTALLATION PROBLEM

### DISCUSSION THREAD

**Discussion by MAG-ENCRYPTION** [community/community#40351] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

When installing the NFLlib encryption library I installed Cmake and the others without problems but I encountered a problem when I executed the line below on my linux machine
$> cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$HOME/nfllib
  I am shown an error.
I even tried changing the path to no avail.

**Comment by ossanna16** [comment]
Hi there @MAG-ENCRYPTION and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
